### PR TITLE
feat: allow 47 to sit on a chair from the Undying pack without enabling the matching living room cosmetics

### LINKED ISSUE #228: Kronstadt Chair from the Undying pack cannot be sat on unless you have the associated living room chairs also enabled
if you had the Undying pack, 47 could not sit on the chair in the corner from that pack if he didn't have the associated living room cosmetics also applied.

screenshot_8118

The issue is that the top value show in the image is from the lounge set, likely left in by mistake by the developers. This patch removes that value and leaves only the one for the kronstadt chair.
screenshot_8114

### PR DESCRIPTION
**Pull request by dribbleondo**
fixes #228

### CODE REVIEW

**Comment by Ocean-Minnow** [comment]
In the future, it would be much better to detail the fix in the PR rather than the issue, so that it can be reviewed easily on one page rather than needing to open two.

I'll test this out when I get the chance. Or someone else will :p

File: content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+	"tempHash": "00A9B93A825F5BDC",
+	"tbluHash": "005BA5CA6AC00627",
+	"patch": [
+		{
+			"SubEntityOperation": [
+				"0e4a8142f0ad2c47",
+				{
+					"SetPropertyValue": {
+						"property_name": "m_mTransform",
```
Comment (Ocean-Minnow) [inline content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json:10]: Are these two Transforms necessary for enabling the interaction?

Comment (VoodooHillbilly) [reply content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json:10]: No. They're both unrelated items that GlacierKit marks as changed just by clicking on them.

Comment (dribbleondo) [reply content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json:10]: Yep that sounds about right.

Comment (Ocean-Minnow) [reply content/SmallFixes/chunk29/kronstadt_chair_fix/kronstadt_chair_you_have_researched_sitting_fix.entity.patch.json]: Ah yeah, that's happened to me before.

**Comment by VoodooHillbilly** [comment]
I'd recommend dropping the middle part of the filename so it becomes `kronstadt_chair_sitting_fix.entity.patch.json`

**Comment by dribbleondo** [comment]
Done.

**Comment by VoodooHillbilly** [comment]
LGTM.

**Pull request merged**